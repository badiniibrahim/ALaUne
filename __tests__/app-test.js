import React from "react";
import renderer from 'react-test-renderer';
import App from '../app/app';

test('Testing app snapshot', () => {
    const snap = renderer.create(
        <App />
    ).toJSON();

expect(snap).toMatchSnapshot();
})