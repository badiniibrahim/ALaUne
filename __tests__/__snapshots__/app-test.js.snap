// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing app snapshot 1`] = `
<View>
  <Text>
    Hello Badini !
  </Text>
</View>
`;
